tasks/post_copy_request_to_queue/test/unit_tests/test_post_copy_request_to_queue.py

```diff
@@ -342,7 +340,7 @@ def test_task_happy_path(
                 post_copy_request_to_queue.TARGET_KEY_KEY: key_path,
                 post_copy_request_to_queue.SOURCE_BUCKET_KEY: bucket_name,
             }
-        ]
+        mock_query_db.return_value = [row]
```

Comment:
We have cases where lambdas modify input before using it, so that's why I did this excessive multi-initiation. This way is better so long as we do`mock_query_db.return_value = [copy.deepcopy(row)]
See line 410 for an example.

Reply:
ok used mock_query_db.return_value = [copy.deepcopy(row)]